webroot/faq/locales/faq.ja.md

```diff
@@ -141,13 +141,13 @@ A: Chia Networkは、空いているストレージ（例えばHDD）領域を
 
 + <b>Q: Chiaはオープンソースですか?</b>
 
-A: ブロック形成に関する学術論文を発表し、すべての実装コードはオープンソースになります。 私たちは<a href = "https://www.apache.org/licenses/LICENSE-2.0">Apache License</a>を使用します。テストネットだけがあってメインネットがまだない時には、コードがプロプライエタリである場合がありますが、メインネットが稼働を始まったら、確実にAppachライセンスで公開します。
+A: ブロック形成に関する学術論文を発表し、すべての実装コードはオープンソースになります。 私たちは<a href = "https://www.apache.org/licenses/LICENSE-2.0">Apache License</a>を使用します。テストネットだけがあってメインネットがまだない時には、コードがプロプライエタリである場合がありますが、メインネットが稼働を始まったら、確実にAppacheライセンスで公開します。
```

Comment:
Correct spell is `Apache`, there's one too many `p`.

Reply:
Oh... Sorry to bother you again.